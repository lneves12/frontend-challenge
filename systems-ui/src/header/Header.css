.brandText {
  padding-left: 5px;
}
