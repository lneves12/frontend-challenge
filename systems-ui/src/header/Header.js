import React, {Component} from "react";
import suseLogo from "./suse.svg";
import styles from "./Header.css"

class Header extends Component {
  render() {
    return (
        <nav className="navbar navbar-dark bg-dark">
          <div className="navbar-brand">
            <img width="50" src={suseLogo} alt="logo" />
            <span className={styles.brandText}> Systems Dashboard</span>
          </div>
        </nav>
    );
  }
}

export default Header;
