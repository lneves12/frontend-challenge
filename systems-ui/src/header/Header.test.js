import React from 'react';
import Header from './Header';
import renderer from 'react-test-renderer';
import {MemoryRouter} from "react-router";

it('renders correctly', () => {
  const tree = renderer.create(
    <MemoryRouter>
      <Header />
    </MemoryRouter>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
