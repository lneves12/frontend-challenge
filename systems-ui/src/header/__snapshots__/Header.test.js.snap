// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="navbar navbar-dark bg-dark"
>
  <div
    className="navbar-brand"
  >
    <img
      alt="logo"
      src="suse.svg"
      width="50"
    />
    <span
      className={undefined}
    >
       Systems Dashboard
    </span>
  </div>
</nav>
`;
